components/progressbar/index.css

```diff
@@ -127,6 +127,20 @@
 	&.is-negative .spectrum-ProgressBar-fill {
 		background-color: var(--highcontrast-progressbar-fill-color, var(--mod-progressbar-fill-color-negative, var(--spectrum-progressbar-fill-color-negative)));
 	}
+
+	&.spectrum-ProgressBar--staticWhite {
+		&.is-positive .spectrum-ProgressBar-fill {
+			background: var(--mod-progressbar-fill-color-white, var(--spectrum-progressbar-fill-color-white));
+		}
+
+		&.is-notice .spectrum-ProgressBar-fill {
+			background: var(--mod-progressbar-fill-color-white, var(--spectrum-progressbar-fill-color-white));
+		}
+
+		&.is-negative .spectrum-ProgressBar-fill {
+			background: var(--mod-progressbar-fill-color-white, var(--spectrum-progressbar-fill-color-white));
+		}
+	}
```

Comment:
I have this all separated, but really the background is exactly the same for all 3 of these variants. Is it too confusing to combine the selectors? 

```
&.is-positive .spectrum-ProgressBar-fill,
&.is-notice .spectrum-ProgressBar-fill,
&.is-negative .spectrum-ProgressBar-fill {
  background: var(--mod-progressbar-fill-color-white, var(--spectrum-progressbar-fill-color-white));
}
```

I guess my only concern is that the `spectrum-ProgressBar--staticWhite` block is already nested, and then these 3 would be nested and stacked. Is that weird from a code readability standpoint at all?